16 define eslint and prettier override sections (#19)
JBKLabs/react-dev: README.md

```json
{
  "scripts": {
    "lint": "jbk-scripts lint",
    "start": "jbk-scripts start",
    "build": "jbk-scripts build",
    "configure": "jbk-scripts configure"
  },
}
```

## Overriding Configuration

### Eslint

In order of precedence, each of the following paths will override the eslint configuration provided by `@jbknowledge/react-dev`

- `jbk-scripts lint --eslint-config <rootRelativePath>`
- `.eslintrc`
- `.eslintrc.json`
- `.eslintrc.yml`
- `.eslintrc.yaml`
- `package.json#eslintConfig`

You can reference the [eslint user guide](https://eslint.org/docs/user-guide/configuring) for how to do so.

> As a peer dependency, `@jbknowledge/eslint-config`, which `@jbknowledge/react-dev` uses implicitly, you are able to extend the existing eslint configuration like you would for any other project via `extends`. For example:

Reviewer: This sentence is confusing, I would suggest re-wording.

Reviewer: ```suggestion
`@jbknowledge/react-dev` uses `@jbknowledge/eslint-config` implicitly. You can extend this configuration like you would for any other project via `extends`. For example:
```
